### DISCUSSION TITLE
ppocr的识别字典

### DISCUSSION THREAD

**Discussion by cloudfool** [PaddlePaddle/PaddleOCR#12370] [Q&A]
1.ppocr采用的字典文件就是ppocr_kyes_v1，我发现有很多常用字不在这个文件中啊，像“珅”"琀"等，这个如何解决？
2.我发现ppocr倾向于识别成更常用的字，比如”昊“容易识别成”吴“，”馕“容易识别成”镶“，反过来则基本不会发生，求指导解答一下。
谢谢

**Comment by cloudfool** [comment]
我感觉有些字虽然在ppocr_keys_v1里存在，但是模型好像总是识别不出..

**Comment by TingquanGao** [comment]
目前ppocr只提供了适用于大部分日常场景的文档识别，如果您有私有数据，可以尝试在我们现有模型的基础上fine-tune。
另外，我们在PaddleX的PP-ChatOCRV2中集成了大字典模型，支持1.5万的字库，可以尝试下：

https://aistudio.baidu.com/application/detail/10368

![9d601dadb642a8de5de0b7c29947eea3](https://github.com/PaddlePaddle/PaddleOCR/assets/12560511/fb745f52-37d6-4583-bfce-ed2086a2e1b5)
